client/client.go

```diff
@@ -117,6 +122,30 @@ func (c *client) TransactionReceipts(ctx context.Context, txs types.Transactions
 	return receipts, nil
 }
 
+func (c *client) UncleByBlockHashAndPosition(ctx context.Context, hash common.Hash, position uint) (*types.Header, error) {
+	var result *types.Header
+	err := c.rpc.CallContext(ctx, &result, "eth_getUncleByBlockHashAndIndex", hash, hexutil.Uint(position))
+	if err == nil && result == nil {
```

Comment:
Can you also add this check in other client api?

Reply:
OK